src/content/software/cornell.mdx

```diff
@@ -4,7 +4,7 @@ company: "Cornell University"
 startDate: 2016-09-01
 endDate: 2020-05-15
 description: "I obtained my undergraduate degree at Cornell University"
-logo: "./cornell_seal_simple_web.svg"
+logo: "./cornell_seal_simple_web.png"
```

Comment:
I was having trouble getting the SVG to render nicely so I switched it out for a png for now